databox/client/Dockerfile

```diff
@@ -16,6 +16,7 @@ COPY --from=client-build /usr/src/app/build /var/app
 COPY docker/nginx/conf.d /etc/nginx/conf.d
 
 RUN apk add --no-cache libstdc++ \
+    && apk --no-cache add perl \
```

Comment:
Why did you add `perl` to the client? Did you mean `databox/api/Dockerfile` instead?